src/06_wildfire.R

```diff
@@ -0,0 +1,569 @@
+
+# Available Resources (1):
+# 1. View WMS getCapabilities request details (wms)
+# Access the full 'Resources' data frame using:
+bcdc_tidy_resources('cdfc2d7b-c046-4bf0-90ac-4897232619e1')
+# Query and filter this data using: bcdc_query_geodata('cdfc2d7b-c046-4bf0-90ac-4897232619e1')
+
+BC_wildfire_perimeter_current <-  bcdc_query_geodata('cdfc2d7b-c046-4bf0-90ac-4897232619e1') %>%
+  filter(FIRE_YEAR >= 2024) %>% # The data in BC data has one row FIRE_NUMBER == 'G51564' for 2023 which is duplicated since it is in historic dataset.
+  collect() %>%
+  mutate(FIRE_LABEL = paste(FIRE_YEAR, FIRE_NUMBER, sep = '-'))
+# 
+# file_path <- use_network_path("data/Wildfires_DB/Input/BC Wildfire Fire Perimeters - Current GEOJSON/PROT_CURRENT_FIRE_POLYS_SP.geojson")
+# BC_wildfire_perimeter_current <- st_read(file_path) %>%
+#   filter(FIRE_YEAR>=2024) %>%
+#   mutate(FIRE_LABEL = paste(FIRE_YEAR, FIRE_NUMBER, sep='-'))
+
+print(st_crs(BC_wildfire_perimeter_current))
+
+# The same as the historic wild fire data
+
+# BC_wildfire_perimeter_current %>% 
+#   glimpse()
+
+# BC_wildfire_perimeter_current %>% 
+#   group_by(FIRE_LABEL) %>% 
+#   filter(n()>1)
+# no duplicated FIRE_LABEL, no need for dealing with duplication like historic dataset.
+
+# Columns unique to df1
+# unique_to_df1 <- setdiff(colnames(BC_wildfire_perimeter_historic), colnames(BC_wildfire_perimeter_current))
+# 
+# # Columns unique to df2
+# unique_to_df2 <- setdiff(colnames(BC_wildfire_perimeter_current), colnames(BC_wildfire_perimeter_historic))
+# 
+# # Common columns
+# common_columns <- intersect(colnames(BC_wildfire_perimeter_historic), colnames(BC_wildfire_perimeter_current))
+# 
+# # Print differences
+# print(unique_to_df1)  # ["a"]
+# print(unique_to_df2)  # ["c"]
+# print(common_columns) # ["b"]
+
+
+# Bind rows
+BC_wildfire_perimeter = bind_rows(BC_wildfire_perimeter_historic, BC_wildfire_perimeter_current) %>% 
+  arrange(FIRE_LABEL) 
+
+# BC_wildfire_perimeter %>% 
+#   glimpse()
+
+# BC_wildfire_perimeter %>%
+#   group_by(FIRE_LABEL) %>%
+#   filter(n()>1)
+
+###################################################################
+# # boundary files
+# https://www12.statcan.gc.ca/census-recensement/2021/geo/sip-pis/boundary-limites/index2021-eng.cfm?year=21
+# 
+# # geographic attribute file
+# https://www12.statcan.gc.ca/census-recensement/2021/geo/aip-pia/attribute-attribs/index-eng.cfm
+###################################################################
+
+# grab the file from "Statistics Canada", only run once.
+# download.file("https://www12.statcan.gc.ca/census-recensement/2021/geo/sip-pis/boundary-limites/files-fichiers/ldb_000a21a_e.zip",
+#               destfile=use_network_path("data/Wildfires_DB/Input/Blocks_2021/ldb_000a21a_e.zip"))
+# unzip(use_network_path("data/Wildfires_DB/Input/Blocks_2021/ldb_000a21a_e.zip"))
+## READ DISSEMINATION BLOCKS
+file_path <- use_network_path("data/Wildfires_DB/Input/Blocks_2021/ldb_000b21a_e.shp")
```

Comment:
File has moved:

`file_path <- use_network_path("data/raw_data/Wildfire/Wildfires_DB/Input/Blocks_2021/ldb_000b21a_e.shp")`

Reply:
Thanks for catching that. Accepted the change.